src/CustomElementRegistry.js

```diff
@@ -47,7 +47,7 @@ export default class CustomElementRegistry {
      * @private
      * @type {!Array<string>}
      */
-    this._unflushedLocalNames = [];
+    this._unflushedNames = [];
```

Comment:
Similar to `[localNameToDefinition](https://github.com/webcomponents/custom-elements/pull/88/files#diff-4b635b4d15236896a8e79c481a320662R32)`, we now need to keep track of things based on the custom element name, not on the local name. This is because customized builtins have a different localName than a CE name.